src/widgets/panel/panel.tsx

```diff
@@ -202,16 +211,16 @@ const panel: PanelWidget = panelWidgetParams => {
   };
 
   return widgetFactory => widgetParams => {
-    const { container } = widgetParams || {};
-
-    if (!container) {
+    if (!(widgetParams && widgetParams.container)) {
```

Comment:
```suggestion
    if (!widgetParams || !widgetParams.container) {
```

not too different, but reads easier, IMO.